.github/workflows/linux-pipeline.yml

```diff
@@ -112,7 +107,7 @@ jobs:
           "$GITHUB_SERVER_URL/$GITHUB_REPOSITORY"
 
         ELECTRON_USE_THREE_WAY_MERGE_FOR_PATCHES=1 gclient sync --with_branch_heads --with_tags -vvvvv
-        if [ "${{ inputs.is-release }}" != "true" ]; then
+        if [ ${{ inputs.is-release != true }}]; then
```

Comment:
@codebytere had an issue with this, we should use `=` or `!=` outside of the templating. i.e. the original thing is better here